flink-connector-pulsar-e2e-tests/src/test/java/org/apache/flink/tests/util/pulsar/common/FlinkContainerUtils.java

```diff
@@ -35,9 +35,11 @@ public class FlinkContainerUtils {
     public static Configuration flinkConfiguration() {
         Configuration configuration = new Configuration();
 
-        // Increase the jvm metaspace memory to avoid java.lang.OutOfMemoryError: Metaspace
-        configuration.set(TaskManagerOptions.TOTAL_PROCESS_MEMORY, MemorySize.ofMebiBytes(2048));
+        configuration.set(TaskManagerOptions.TOTAL_PROCESS_MEMORY, MemorySize.ofMebiBytes(2560));
+        configuration.set(TaskManagerOptions.TASK_OFF_HEAP_MEMORY, MemorySize.ofMebiBytes(512));
         configuration.set(TaskManagerOptions.JVM_METASPACE, MemorySize.ofMebiBytes(512));
+
+        // Increase the jvm metaspace memory to avoid java.lang.OutOfMemoryError: Metaspace
         configuration.set(JobManagerOptions.TOTAL_PROCESS_MEMORY, MemorySize.ofMebiBytes(2560));
         configuration.set(JobManagerOptions.JVM_METASPACE, MemorySize.ofMebiBytes(1024));
```

Comment:
```suggestion
        configuration.set(TaskManagerOptions.TOTAL_PROCESS_MEMORY, MemorySize.ofMebiBytes(2560));
        configuration.set(TaskManagerOptions.TASK_OFF_HEAP_MEMORY, MemorySize.ofMebiBytes(512));
        // Increase the jvm metaspace memory to avoid java.lang.OutOfMemoryError: Metaspace
        configuration.set(TaskManagerOptions.JVM_METASPACE, MemorySize.ofMebiBytes(512));

      
        configuration.set(JobManagerOptions.TOTAL_PROCESS_MEMORY, MemorySize.ofMebiBytes(2560));
        // Increase the jvm metaspace memory to avoid java.lang.OutOfMemoryError: Metaspace
        configuration.set(JobManagerOptions.JVM_METASPACE, MemorySize.ofMebiBytes(1024));
```

Reply:
Is this more appropriate(perhaps)? But it doesn't matter if we don't make any change. 😆